File: backend/model/entities/post_usuario.go
package entities

import "time"

type PostUsuario struct {
	PostUserId          uint      `json:"post_user_id"`
	Descripcion         string    `json:"descripcion"`
	UsuarioId           string    `json:"usuario_id"`
	File                string    `json:"file"`
	UsuarioCreacion     string    `json:"usuario_creacion"`
	UsuarioModificacion string    `json:"usuario_modificacion"`
	FechaCreacion       time.Time `json:"fecha_creacion"`
	FechaModificacion   time.Time `json:"fecha_modificacion"`
}

File: frontend/src/models/post-usuario.ts

export interface PostUsuarioDTO {
    post_user_id: number;
    descripcion: string;
    usuario: string;
    usuario_id: number ;
    id_storage: number;
    url: string;
    usuario_creacion: string;
    usuario_modificacion: string;
    fecha_creacion: string;
    fecha_modificacion: string;
}

export interface PostUsuario {
    post_user_id: number
    descripcion: string
    usuario_id: number | null
    file: ArrayBuffer | null
    fileName: string
    usuario_creacion: string
    usuario_modificacion: string
}

